### PR TITLE
Add Dataminer Health Check Tool

### PR DESCRIPTION
**Pull request by Daniela-Prada**

### CODE REVIEW

File: user-guide/images/Health_Check_Email_Report.png
Comment (MariekeGO) [inline user-guide/images/Health_Check_Email_Report.png:1]: This image is a bit blurry. Could you replace it with a higher-quality image?

File: user-guide/Reference/DataMiner_Tools/Dataminer_Health_Check/Dataminer_Health_Check_Tool_Configuration.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+   ![Add Test option](~/user-guide/images/Health_Check_Add_Test.png)
+
+1. In the pop-up window, fill in the required fields:
+
+   - *Test Type*: Make sure *Script* is selected.
+   - *Name*: Select an available script-based test.
+   - *Operator* and *Threshold*: Define the conditions for the test to determine whether it passes or fails.
+   - *Display Title*: Optionally, enter a custom name for the test to make it easily identifiable. If provided, this title will be used to identify the test in the email report; otherwise, the default test name will be displayed.
+   - *Element/DMA Filter*: Use this field to apply the test to a specific element. Enter the element/DMA you wish to target.
+   - *Display Key Filter*: Apply this filter if you want the test to consider only certain entries for the test.
```
Comment (MariekeGO) [inline user-guide/Reference/DataMiner_Tools/Dataminer_Health_Check/Dataminer_Health_Check_Tool_Configuration.md:40]: Which format should be used for this? Are wildcards supported?

File: user-guide/Reference/DataMiner_Tools/Dataminer_Health_Check/Dataminer_Health_Check_Tool_Configuration.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+   ![Add Test option](~/user-guide/images/Health_Check_Add_Test.png)
+
+1. In the pop-up window, fill in the required fields:
+
+   - *Test Type*: Make sure *Script* is selected.
+   - *Name*: Select an available script-based test.
+   - *Operator* and *Threshold*: Define the conditions for the test to determine whether it passes or fails.
+   - *Display Title*: Optionally, enter a custom name for the test to make it easily identifiable. If provided, this title will be used to identify the test in the email report; otherwise, the default test name will be displayed.
+   - *Element/DMA Filter*: Use this field to apply the test to a specific element. Enter the element/DMA you wish to target.
```
[the remark this answers is not in the document]
Comment (Daniela-Prada) [reply user-guide/Reference/DataMiner_Tools/Dataminer_Health_Check/Dataminer_Health_Check_Tool_Configuration.md:39]: Yes ElementID/DmaId

[the remark this answers is not in the document]
Comment (Daniela-Prada) [reply user-guide/Reference/DataMiner_Tools/Dataminer_Health_Check/Dataminer_Health_Check_Tool_Configuration.md:39]: It should actually be the name of the element

File: user-guide/Reference/DataMiner_Tools/Dataminer_Health_Check/Dataminer_Health_Check_Tool_Configuration.md (hunk 1 above)
Comment (Daniela-Prada) [reply user-guide/Reference/DataMiner_Tools/Dataminer_Health_Check/Dataminer_Health_Check_Tool_Configuration.md:40]: It should be in the format of a regular expression.

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**